### PR TITLE
Add support for ARM 64 executables on Windows 

### LINKED ISSUE #73: Add support for ARM-64 Launchers
https://discuss.python.org/t/announcement-distlib-0-3-3-released-on-pypi/10763

This should be a matter of running `nox -s update_launchers` followed by updating the logic to accept and use the correct launcher.

### PR DESCRIPTION
**Pull request by FFY00**
Fixes #73

Signed-off-by: Filipe Laíns <lains@riseup.net>

### CODE REVIEW

**Comment by pradyunsg** [comment]
That binary does not look right to me.

**Comment by FFY00** [comment]
Oops. Fixed, also added a test to verify the signature.

File: tests/test_scripts.py [hunk 1]
```diff
@@ -40,6 +40,15 @@ def _read_launcher_data(section, kind):
         return f.read()
 
 
+@pytest.mark.parametrize("launcher", sorted(_ALLOWED_LAUNCHERS.values()))
```
[the remark this answers is not in the document]
Comment (pradyunsg) [reply tests/test_scripts.py:43]: I don't think this test is necessary at all.

The executables are manually updated, and the mistake of getting the executable name wrong is a one-off that doesn't need additional protections; especially given that it involves sharing an underscored name across modules.

**Review by pradyunsg — APPROVED** [review]

**Comment by pradyunsg** [comment]
Thanks @FFY00 for picking this up, and shoutout to @vsajip for all of their tireless work on distlib that we're building off of here. :)

File: tests/test_scripts.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (FFY00) [reply tests/test_scripts.py]: We could at least add the check to the downloader script, but your call.

**Pull request merged**